doc/api/zlib.md

```diff
@@ -460,7 +460,7 @@ added: v0.6.0
 added: v0.11.12
 -->
 
-Compress a Buffer or string with Deflate.
+Compress a Buffer or string with [Deflate][].
```

Comment:
@danbev I don’t feel very strongly about it, so I’ve gone ahead and added links for the `Buffer` occurrences.